src/fmu/dataio/dataio.py

```diff
@@ -192,9 +199,29 @@ def _content_validate(name, fields):
                     f"Invalid type for <{key}> with value <{dtype}>, not of "
                     f"type <{wanted_type}>"
                 )
+        elif DEPRECATED_CONTENTS.get(name, {}).get(key, None) is not None:
+            logger.debug("%s/%s is deprecated, issue warning", name, key)
+            replaced_by = DEPRECATED_CONTENTS[name][key].get("replaced_by", None)
+
+            message = f"Content {name}.{key} is deprecated. "
+
+            if replaced_by is not None:
+                message += f"Please use {replaced_by}. "
+                replace_deprecated.update({key: replaced_by})
+
+            warn(
+                message,
+                PendingDeprecationWarning,
```

Comment:
Better with just `DeprecationWarning`? Do not see the ned for "pending" here.

Reply:
Copy/paste... I agree, will fix 👍🏻